test_runner/fixtures/benchmark_fixture.py

```diff
@@ -80,7 +80,13 @@ def parse_from_stdout(
     ):
         stdout_lines = stdout.splitlines()
 
+        number_of_clients = 0
+        number_of_threads = 0
+        number_of_transactions_actually_processed = 0
+        latency_average: float = 0
         latency_stddev = None
+        tps: float = 0
```

Comment:
Optionally, if this works, maybe for consistensy: 
```suggestion
        tps = 0.0
```